ftw/labels/migrations/migrate.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```

Comment:
We have a convention that we usually place upgrades in a directory called `upgrades` in our packages (no way you could have known, since there aren't any upgrades yet). So could you please rename this file to  `ftw/labels/upgrades/upgrade_to_2001.py` (and update the `configure.zcml` accordingly)?